sdk/core/core/inc/az_credentials.h

```diff
@@ -7,15 +7,12 @@
 #include <az_http.h>
 #include <az_result.h>
 #include <az_span.h>
+#include <az_constants_internal.h>
 
 #include <stdint.h>
 
 #include <_az_cfg_prefix.h>
 
-enum {
-  _az_TOKEN_MAX_LENGTH = 2 * 1024,
```

Comment:
guess this was different than url size.  This is credential only config.
max_url needs to be removed from http.h